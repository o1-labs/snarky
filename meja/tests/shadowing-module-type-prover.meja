module type M = {
  type t;
};

Prover {
  module type M = {
    type u;
  };
};
