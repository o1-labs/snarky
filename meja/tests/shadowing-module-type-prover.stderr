File "tests/shadowing-module-type-prover.meja", line 6, characters 14-15:
Error: There is already a module type with name M.
